packages/logs/src/domain/logsCollection/rawError/rawErrorCollection.ts

```diff
@@ -0,0 +1,53 @@
+import type { Context, RawError, ClocksState } from '@datadog/browser-core'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I feel that `rawError` is a bit vague: it doesn't indicate what is collected. What about:
* moving `reportRawError` into a dedicated utility module, and change it to take `sender` as argument
* changing the `trackNetworkError` to follow the `consoleCollection` and `reportCollection` pattern:
  * renaming it to `networkErrorCollection`
  * take a `configuration` and `sender` as argument
  * instead of formating `rawErrors`, directly use `sender` to send errors: one less intermediate format!
  * in `startLogs`, call `startNetworkErrorCollection`
* implementing a new `runtimeErrorCollection` following the `consoleCollection` and `reportCollection` pattern:
  * uses `trackRuntimeError`
  * use the newly created `reportRawError` function to report raw errors 
  * in `startLogs`, call `startRuntimeErrorCollection`
* assembly could also take `sender` as argument and use the `reportRawError` module